src/openrct2/actions/FootpathPlaceFromTrackAction.cpp

```diff
@@ -43,6 +43,16 @@ void FootpathPlaceFromTrackAction::Serialise(DataSerialiser& stream)
            << DS_TAG(_constructFlags);
 }
 
+void FootpathPlaceFromTrackAction::AcceptParameters(GameActionParameterVisitor& visitor)
```

Comment:
Is that a good reason? I can already see a use case for Scenery Manager and other plugins, e.g. to place exact queue layouts, which is not possible (or rather unneccessarily difficult) with the regular action.

Reply:
if you feel it has a use. But does it make sense for it to be found under the `FootpathPlaceFromTrackAction` name.

Reply:
I tried to keep the names as close to the originals while making the sure that the naming conventions are consistent. But you are right, from the viewpoint of the API, the difference is not that you place it wrt some track, but that you specify the edges instead of having them inferred. The name should capture this, do you have a suggestion?
Maybe FootpathPlaceWithLayout or FootpathLayoutPlace.

Reply:
Your call on the names I don't have any preference.